### PR TITLE
Delete transformer

### PR DESCRIPTION
**Pull request by pruksmhc**
https://github.com/nyu-mll/jiant/issues/632

### CODE REVIEW

File: src/trainer.py [hunk 1]
```diff
@@ -96,13 +96,11 @@ def build_trainer(
     phase="pretrain",
 ):
     """Build a trainer from params.
-
     Parameters
     ----------
     params: Trainer parameters as built by build_trainer_params.
     model: A module with trainable parameters.
     run_dir: The directory where we save the models.
-
```
Comment (sleepinyourhat) [inline src/trainer.py]: Why cut these?

File: src/trainer.py [hunk 2]
```diff
@@ -895,14 +885,12 @@ def _validate(self, epoch, tasks, batch_size, periodic_save=True):
         Validate on all tasks and return the results and whether to save this epoch or not.
         Note/TODO: 'Epoch' here refers to validation passes, not proper epochs over
         any given task's training set.
-
```
Comment (pruksmhc) [inline src/trainer.py]: This isn't deleted - not sure why github is showing that it is.

Comment (sleepinyourhat) [reply src/trainer.py]: Probably out of sync with master...

File: src/trainer.py [hunk 3]
```diff
@@ -121,29 +121,17 @@ def build_trainer(
         opt_params["warmup"] = 0.1
```
Comment (sleepinyourhat) [inline src/trainer.py]: A comment a couple of lines above mentions transformer. Double check that it still applies to BERT or GPT.

Comment (pruksmhc) [reply src/trainer.py]: Yep it has to do with BERT.

**Pull request merged**